src/celeritas/ext/RootUniquePtr.hh

```diff
@@ -0,0 +1,70 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2022-2023 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/ext/RootUniquePtr.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <memory>
+
+#include "celeritas_config.h"
+#include "corecel/Assert.hh"
+
+// Forward-declare ROOT
+class TObject;
```

Comment:
Since nothing here uses TObject we can delete this... we're basically back to where we were before where these are "generic" deleters.